src/doc/src/reference/manifest.md

```diff
@@ -73,12 +73,17 @@ The package name is an identifier used to refer to the package. It is used
 when listed as a dependency in another package, and as the default name of
 inferred lib and bin targets.
 
-The name must use only [alphanumeric] characters or `-` or `_`, and cannot be empty.
+The name must use only [alphanumeric] characters or `-` or `_`, and cannot be
+empty.
```

Comment:
(not a blocker)
Could we not wrap this? ehuss prefers to [semantic linefeeds](https://github.com/rust-lang/cargo/pull/10916#discussion_r934132791) which I bought into as well 😆

Reply:
Of course!